lib/utils/localization.js

```diff
@@ -1,24 +1,44 @@
 /* @flow */
 
-import moment from 'moment';
+import { once } from 'lodash/fp';
 import { locale } from '../environment';
 
+// Avoid initializing dayjs before it's necessary
+const initialize = once(() => {
+	require('dayjs/locale/de'); // eslint-disable-line global-require
+	require('dayjs/locale/el'); // eslint-disable-line global-require
+	require('dayjs/locale/es'); // eslint-disable-line global-require
+	require('dayjs/locale/he'); // eslint-disable-line global-require
+	require('dayjs/locale/it'); // eslint-disable-line global-require
+	require('dayjs/locale/nl'); // eslint-disable-line global-require
+	require('dayjs/locale/pl'); // eslint-disable-line global-require
+	require('dayjs/locale/pt-br'); // eslint-disable-line global-require
+	require('dayjs/locale/pt'); // eslint-disable-line global-require
+	const dayjs = require('dayjs'); // eslint-disable-line global-require
+	dayjs.extend(require('dayjs/plugin/relativeTime')); // eslint-disable-line global-require
+	dayjs.extend(require('dayjs/plugin/localizedFormat')); // eslint-disable-line global-require
+	dayjs.locale(locale());
```

Comment:
Hmm, since we can get the locale via `locale`, maybe we can replace all of the `require` lines with a single <code>require(&#96;dayjs/locale/${locale()}&#96;);</code> call?

Reply:
I don't think Webpack will be able to resolve that without also bundling all locales.